src/runtime/components/forms/RadioGroup.vue

```diff
@@ -86,11 +88,16 @@ export default defineComponent({
     ui: {
       type: Object as PropType<Partial<typeof config & { strategy?: Strategy }>>,
       default: undefined
+    },
+    uiRadio: {
+      type: Object as PropType<Partial<typeof configRadio & { strategy?: Strategy }>>,
+      default: undefined
     }
   },
   emits: ['update:modelValue', 'change'],
   setup (props, { emit }) {
     const { ui, attrs } = useUI('radioGroup', toRef(props, 'ui'), config, toRef(props, 'class'))
+    const { ui: radioUI } = useUI('radio', toRef(props, 'uiRadio'), configRadio)
```

Comment:
I'd rather call it `uiRadio` to keep consistency with `SelectMenu` and others: https://github.com/nuxt/ui/blob/dev/src/runtime/components/forms/SelectMenu.vue#L319

Reply:
Not sure why I didn't change that to match the prop 😅